zanata-war/src/main/webapp/WEB-INF/pages.xml

```diff
@@ -218,7 +203,6 @@
   </page>
 
   <page view-id="/admin/usermanager.xhtml" action="#{userSearch.loadUsers}">
-    <restrict>#{s:hasPermission('seam.user', 'read')}</restrict>
```

Comment:
All the user management work in UserAction is delegated to ZanataIdentityManager which will check all the security in its methods. But I do forgot to secure userSerach.loadUsers. it should also delegate to ZanataIdentityManager not the JpaIdentityStore directly. 
----- Original Message -----

> From: "Sean Flanigan" <notifications@github.com>
> To: "zanata/zanata-server" <zanata-server@noreply.github.com>
> Cc: "Patrick Huang" <pahuang@redhat.com>
> Sent: Friday, July 31, 2015 3:16:49 PM
> Subject: Re: [zanata-server] Replace Seam Security (#908)

> In zanata-war/src/main/webapp/WEB-INF/pages.xml :
> > @@ -218,7 +203,6 @@
> >    </page>
> >
> >    <page view-id="/admin/usermanager.xhtml"
> >    action="#{userSearch.loadUsers}">
> > -    <restrict>#{s:hasPermission('seam.user', 'read')}</restrict>

> I can't find this security check in the UserAction bean.

> —
> Reply to this email directly or view it on GitHub .